src/hydra-eval-jobs/hydra-eval-jobs.cc

```diff
@@ -416,7 +416,11 @@ int main(int argc, char * * argv)
 
                     if (response.find("attrs") != response.end()) {
                         for (auto & i : response["attrs"]) {
-                            auto s = (attrPath.empty() ? "" : attrPath + ".") + (std::string) i;
+			    std::string path = i;
+			    if (path.find(".") != std::string::npos){
+				    path = "\"" + path  + "\"";
+			    }
```

Comment:
mixed indent

Reply:
Thanks :)

Reply:
Should be spaces, but is all tabs now.

Reply:
Hopefully this is good now ! We should maybe recommend a formatter in the readme or provide a treefmt.toml file.